pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/extension/PythonVersion.java

```diff
@@ -21,6 +21,10 @@
 
     private final String version;
 
+    public static String[] whitelistedPythonVersions = { "2.6", "2.7", "3.4", "3.5", "3.6" };
+    public static String defaultPython2 = "2.6";
+    public static String defaultPython3 = "3.5";
```

Comment:
@zvezdan and I discussed some API changes, which I think make sense.  `PythonVersion` itself isn't really exposed so we'll make those non-`static` and expose an API on `PythonDetails` to get and set the default versions and whitelist.  That might address both use cases better.